dea_check.py

```diff
@@ -57,10 +42,9 @@
                      'TSCPOS': [(1, 2.5),
                                 (99, 2.5)]
                      }
+HIST_LIMIT = [20.]
```

Comment:
Actually, when I said "check code" I didn't mean the generalized model check code, I just meant do these limits belong in the code part of the dea_check tool (dea_check.py) or should they be pulled out into a config/characteristics file like dea_config.py/dea_config.json or some such.  I have a tendency to "over-configure-ate", but at the same time it can be a nice way to see with your future changes if a change was strictly a limit/characteristics/configuration change or if required a change in the processing logic (a code change).  But, it also makes it harder to see what your limits *are* and where they came from when you are working in the code.